### PR TITLE
Fix encoding and update versionDate field in global.props

### PR DESCRIPTION
**Pull request by kamilszostak**

### CODE REVIEW

File: changeSdkVersion.ps1 [hunk 1]
```diff
@@ -24,13 +24,18 @@ if (-Not ($version -match "\d+\.\d+\.\d+")) {
 
 # update package.json 
 $packagesJson.version = $version
-$packagesJson | ConvertTo-Json | Out-File $packageJsonPath
+$packagesJson = $packagesJson | ConvertTo-Json 
+$packagesJson = $packagesJson -replace "\\u0026", "&"
```
Comment (MaxShehovtsov) [inline changeSdkVersion.ps1]: why is that needed?

Comment (MaxShehovtsov) [inline changeSdkVersion.ps1]: :shipit:

Comment (kamilszostak) [inline changeSdkVersion.ps1]: Json converter is encoding those characters. I need to revert them the original state.
